data/hai/unfettered 1 alphas.txt

```diff
@@ -0,0 +1,1532 @@
+	npc board
+		system "Misam"
+		government "Cerberus"
+		to spawn
+			has "<uhai> 7: Cavalry Arrives: active"
+		personality target launching plunders vindictive uninterested marked staying
+		ship "Bactrian (Cerberus)" "Rested Snake"
+		conversation
+			action
+				set "unfettered: boarded bactrian"
+			`Boarding the ship, you notice that they have other commodities than what you are looking for, but they do have the materials Sookootow talked to you about. Given the damage the ship has suffered, it seems it would require heavy works in the spaceport to get it running again, meaning they will not get repaired anytime soon. You should have some time to switch to a ship with more cargo space, should you need to in order to bring the materials back. You only need 142 tons of Building Materials, take them when you can and bring them back to Sookootow.`
+			branch "allies died"
+				has "<uhai> 7: Cavalry Arrives: failed"
+			`	Your surprise allies hail you. "Well done<, unfettered title>," the first one says. "You have truly impressed us today," the other one adds.`
+			choice
+				`	"Were you sent by Sookootow's contact?"`
+				`	"What are these weapons you are using?"`
+					goto weapons
+			`	They laugh a bit, hearing the way you referenced their leader, "Yes, you could call her that.`
+				goto leave
+			label weapons
+			`	"They are simply a prototype we are testing on the field of battle, as it should be. You will learn more soon enough."`
+			label leave
+			`	"She of the Void does not forget. Nemo Me Impune Lacessit, <last>. Do not presume to attempt to betray us again or there will be consequences."`
+				to display
+					not "midas: tracker relay"
+				goto leaving
+			`	"The Shadow of Mintaka sends her regards, <first> <last>. We look forward to dancing with you again."`
+				to display
+					not "midas: wronged"
+			`	"Do not think we have forgotten of your recent deeds, <first> <last>. But you have proved your worth today."`
+				to display
+					has "midas: wronged"
+			`	You receive a final, private message from one of the ships. "A shame you brought such a fleet with you, we could have had so much fun otherwise... Ra told us you know how to fight."`
+				to display
+					"ships: Space Liner" * 2 + "ships: Transport" * 4 + "ships: Light Freighter" * 2 + "ships: Heavy Freighter" * 4 + "ships: Utility" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 3 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 + "ships: Superheavy" * 18 > 24
+				goto leaving
+			choice
+				`	"Thanks for the assistance."`
+					goto leaving
+				`	"Wait! I challenge you to a duel for those weapons!"`
+			`	They seem surprised, but not disappointed in the slightest, "A challenge, interesting," one says, "We would have much to learn from each others," the other one add. "But what do we have to gain? Our shredders are worth many millions, so I think it would be fair you repaid their value to us, should you lose."`
```

Comment:
```suggestion
			`	They seem surprised, but not disappointed in the slightest, "A challenge, interesting," one says, "We would have much to learn from each other," the other one adds. "But what do we have to gain? Our shredders are worth many millions, so I think it would be fair you repaid their value to us, should you lose."`
```